scripts/test_schemas.sh

```diff
@@ -1,9 +1,15 @@
 #!/usr/bin/env bash
 
-docker pull onsdigital/eq-schema-validator
-validator="$(docker run -d -p 5001:5000 onsdigital/eq-schema-validator)"
+run_docker=true
+if [ "$1" == "-local" ] || [ "$2" == "-local" ]; then
```

Comment:
Could we use double hyphens here? By convention, bash arguments prefixed with a single hypen tend to be single character and can be concatenated (I.e. `ls -al`).

So I'd expect to write `./test_schemas --local`

Reply:
i agree otherwise, this LGTM

Reply:
just gonna test

Reply:
Sorted. 👍